tests/ios/frameworks/genrule-inputs/b/lib.swift

```diff
@@ -0,0 +1,10 @@
+import c
+
+public struct B {
+    public static func run() {
+       C.run()
+       guard LibC != "B" else {
```

Comment:
Sorry If I'm missing something but the only place `LibC` is defined is in `tests/ios/frameworks/genrule-inputs/c/BUILD.bazel` as
```
static NSString *const LibC = @"LibC";
```
so why would you check for `"B"` here?

Reply:
This is just some dummy code to both use the symbol and exercise it for a debugger test in the future.

Reply:
Also it needs to look at `LibC` to make sure that it can load a symbol from a header file defined in `LibC`

Reply:
K thx for clarifying!